### DISCUSSION TITLE
ScriptReference/Collision2D

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#488] [Notes]
# ScriptReference/Collision2D

https://docs.unity3d.com/ScriptReference/Collision2D.html

**Comment by jschiff** [comment]
A couple of the properties on this struct are incredibly confusing. If you are writing an OnCollisionEnter2D and you get one of these structs as a parameter:
- The  `gameObject`, `collider` and `rigidbody` properties refer to the GameObject, Collider2D and Rigidbody2D of the _object you collided with_.
- The `otherCollider` and `otherRigidbody` properties refer to _your own_ Collider2D and Rigidbody2D.
